doc/source/Tutorials/Scripting/CreatingConfigurationFiles3/demos/demoMacbethChart.gfr

```diff
@@ -0,0 +1,154 @@
+__children["Outputs"]["outputs"]["output1"]["parameters"]["displayHost"].addChild( Gaffer.StringPlug( "value", defaultValue = 'localhost', flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["Outputs"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["InteractiveAppleseedRender"] = GafferAppleseed.InteractiveAppleseedRender( "InteractiveAppleseedRender" )
+parent.addChild( __children["InteractiveAppleseedRender"] )
+__children["InteractiveAppleseedRender"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["Catalogue"] = GafferImage.Catalogue( "Catalogue" )
+parent.addChild( __children["Catalogue"] )
+__children["Catalogue"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["latlong_map_environment_edf"] = GafferAppleseed.AppleseedLight( "latlong_map_environment_edf" )
+parent.addChild( __children["latlong_map_environment_edf"] )
+__children["latlong_map_environment_edf"].loadShader( "latlong_map_environment_edf" )
+__children["latlong_map_environment_edf"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["Group"] = GafferScene.Group( "Group" )
+parent.addChild( __children["Group"] )
+__children["Group"]["in"].addChild( GafferScene.ScenePlug( "in1", flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["Group"]["in"].addChild( GafferScene.ScenePlug( "in2", flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["Group"]["in"].addChild( GafferScene.ScenePlug( "in3", flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["Group"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["Camera"] = GafferScene.Camera( "Camera" )
+parent.addChild( __children["Camera"] )
+__children["Camera"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["as_disney_material"] = GafferOSL.OSLShader( "as_disney_material" )
+parent.addChild( __children["as_disney_material"] )
+__children["as_disney_material"].loadShader( "material/as_disney_material" )
+__children["as_disney_material"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["Reference"] = Gaffer.Reference( "Reference" )
+parent.addChild( __children["Reference"] )
+__children["Reference"].load( "/home/michaeldu/dev/gafferDependencies-0.53.0.0-linux/resources/references/macbethTexture.grf" )
```

Comment:
@medubelko References in _.gfr files_ are difficult at the moment. As it stands, Gaffer will save the absolute path to the reference in the file. In addition, at present, sadly there is no way to use variable in the path either. 
I'll create an Issue to add support for this.  As such, I think we have to avoid using Reference in our shipped `.gfr` files until this is resolved.

Reply:
@medubelko @johnhaddon https://github.com/GafferHQ/gaffer/issues/3124

Reply:
@johnhaddon https://github.com/GafferHQ/gaffer/issues/3125